<launch>
    <arg name="queue_size" default="5" />
    <arg name="depth_image" default="/camera/depth/image" />
    <arg name="mono_image" default="/camera/rgb/image_mono" />
    <arg name="camera_info" default="/camera/rgb/camera_info" />
    <arg name="motion_parameters" default="/visual_odometry/motion_matrix" />

    <node pkg="strands_visual_odometry" type="visual_odometry" name="visual_odometry" output="screen">
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="depth_image" value="$(arg depth_image)" type="string"/>
        <param name="mono_image" value="$(arg mono_image)" type="string"/>
        <param name="camera_info" value="$(arg camera_info)" type="string"/>
        <param name="motion_parameters" value="$(arg motion_parameters)" type="string"/>
    </node>
</launch> 
